src/auth/reset_password/reset_password.directive.js

```diff
@@ -0,0 +1,39 @@
+(function () {
+    'use strict';
+
+    angular.module('motech-auth')
+        .directive('resetPassword', ResetPasswordForm);
+
+    function ResetPasswordForm() {
+        return {
+            restrict: 'EA',
+            replace: true,
+            scope: {},
+            controller: 'resetFormController',
+            templateUrl: '/auth/reset_password/reset_password.html',
+            link: Reset_PasswordFormDirective
+        };
+    }
+
+
+
+    function Reset_PasswordFormDirective (scope, element, attrs, controller) {
```

Comment:
Remove redundant empty white lines. There should be a single empty line as a separator.